app/layout/projects/show/map/index.tsx

```diff
@@ -131,18 +134,44 @@ export const ProjectMap: React.FC<ProjectMapProps> = () => {
     },
   });
 
-  const LAYERS = [PUGridLayer, PUCompareLayer, PlanningAreaLayer].filter((l) => !!l);
+  const FeaturePreviewLayers = useFeaturePreviewLayers({
+    features: selectedFeaturesData,
+    active: tab === 'features',
+    bbox,
+    options: {
+      selectedFeatures: selectedFeaturesIds,
+      ...layerSettings['features-preview'],
+    },
+  });
+
+  const selectedPreviewFeatures = useMemo(() => {
+    return selectedFeaturesData
+      ?.filter(({ id }) => selectedFeaturesIds.includes(id))
```

Comment:
You are already filtering by selected feature here: https://github.com/Vizzuality/marxan-cloud/pull/1417/files#diff-a693672f7b0ccf1d136016c3cacf72d2d9636da17aff602db7bf3b2a3cf75ed2R83 You shouldn't need to do it twice.